### PR TITLE
Fix scrub position

### PR DESCRIPTION
**Pull request by edolyne**

### CODE REVIEW

**Comment by johnthepink** [comment]
@edolyne looks good at first glance! Let's step through this in the morning. I'd love to add some inline comments to clarify the changes made. <3

**Comment by edolyne** [comment]
@johnthepink I've gone through and added comments to the changed files. :)

File: imports/components/players/audio/audio.Scrubber.js [hunk 1]
```diff
@@ -50,19 +55,48 @@ export default class AudioScrubber extends Component {
...
+    };
 
     setTimeout(() => {
+      // Set states back to intial values so that time updates,
+      // and click events will happen appropriatly.
+      // Timeout is a full second as a shorter timeout did not
+      // set this states properly.
       this.setState({
-        lastPercent: null,
+        scrubbing: false,
```
Comment (jbaxleyiii) [inline imports/components/players/audio/audio.Scrubber.js]: @edolyne we should add a check to ensure the component is mounted on this. Or add the timeout to the instance and clear it on umount

File: imports/components/players/audio/audio.Scrubber.js [hunk 2]
```diff
@@ -50,19 +55,48 @@ export default class AudioScrubber extends Component {
...
+        lastPercent: percentClicked,
+        override: true
+      });
+
+      this.seek(percentClicked);
+
+      // We need to set the state back to it's initial value to allow incrementing
+      // of the current time.
+      // Timeout is a full second as a shorter timeout did not set the state appropriatly.
+      setTimeout(() => {
```
Comment (jbaxleyiii) [inline imports/components/players/audio/audio.Scrubber.js]: Same thing as above here. setting state when unmounted will throw

**Pull request merged**